classes/auth.php

```diff
@@ -123,12 +123,13 @@ public function __construct() {
         foreach ($idpentities as $idpentity) {
             // Set name.
             $idpentity->name = empty($idpentity->displayname) ? $idpentity->defaultname : $idpentity->displayname;
-            $idpentity->md5entityid = md5($idpentity->entityid);
+            $idpentity->md5id = md5($idpentity->entityid);
+            $idpentity->md5id = md5($idpentity->id);
```

Comment:
Overwriting immediately after?

Reply:
Forgot to cleanup original line, ended up changing that one also, when renaming object key